integration-test/src/main/java/com/scalar/db/api/DistributedStorageMultiplePartitionKeyIntegrationTestBase.java

```diff
@@ -96,14 +106,23 @@ protected int getThreadNum() {
     return THREAD_NUM;
   }
 
-  private void createTables() throws java.util.concurrent.ExecutionException, InterruptedException {
+  protected boolean isParallelDdlSupported() {
+    return true;
+  }
+
+  protected boolean isFloatTypeKeySupported() {
+    return true;
+  }
+
+  private void createTables()
+      throws java.util.concurrent.ExecutionException, InterruptedException, ExecutionException {
     List<Callable<Void>> testCallables = new ArrayList<>();
 
     Map<String, String> options = getCreationOptions();
+    admin.createNamespace(getNamespaceName(), true, options);
```

Comment:
The latency of DDLs is basically very high with YugabyteDB, and many DDLs takes time. I noticed this integration test class didn't need multiple namespaces. So, I changed it to use a single namespace to reduce the duration for testing. Let me know if you have any concern on this change.

Reply:
Ah, got it. The integration test case failed with Cosmos DB due to the limitation `Collection create forbidden as collection count in database offer has exceeded 25` (only 25...?) https://github.com/scalar-labs/scalardb/actions/runs/8906213495/job/24458020986?pr=1710.

I'll revert the change.

Reply:
Done